source/uri/Endpoint.ooc

```diff
@@ -0,0 +1,47 @@
+Endpoint: class {
+	_host: VectorList<Text>
+	_port: Int
+	host: VectorList<Text> {
```

Comment:
Because we only have a getter? Would the syntax be correct if there was a setter below also?